client/components/status/status.js

```
@@ -1250,61 +1355,144 @@ export default class Status extends React.Component {
+                                      </div>
+                                    </CircularProgressbarWithChildren>
+                                  </div>
+                                  <div className="usage-check-used">
+                                    {this.getUserCheckUsedValue(
+                                      check.value,
+                                      check.type,
+                                      check.result,
+                                    )}
+                                  </div>
+                                </div>
+                              ) : null}
```

ESLint

[error] Do not nest ternary expressions.

(no-nested-ternary)